app/code/Magento/Quote/Model/Quote/Item/ToOrderItem.php

```diff
@@ -75,6 +75,7 @@ public function convert($item, $data = [])
         }
 
         $orderItem = $this->orderItemFactory->create();
+        $this->objectCopyService->copyFieldsetToTarget('quote_convert_item', 'to_order_item', $item, $orderItem);
```

Comment:
Similar conversion seems to happen later on in this method. Could you please clarify this change?